VAMobile/src/api/loggedIn/postLoggedIn.tsx

```diff
@@ -0,0 +1,8 @@
+import { post } from 'store/api'
+
+/**
+ * Post user logged in
+ */
+export const postLoggedIn = async () => {
+  await post('v0/user/logged-in')
```

Comment:
Shouldn't this be `/v0/user/logged-in` with a leading slash?

Reply:
good catch on the typo. fixed